rails_programming/advanced_forms_and_activerecord/active_record_queries.md

```diff
@@ -245,3 +240,14 @@ This section contains helpful links to other content. It isn't required, so cons
 * [N+1 Problem: Optimized Counts with Joins and Custom Select](https://www.youtube.com/watch?v=rJg3I-leoo4)
 * [Speed up ActiveRecord with a little tweaking](https://blog.codeship.com/speed-up-activerecord/)
 * [A useful gem that identifies N+1 queries](https://github.com/flyerhzm/bullet)
+
+### Knowledge Check
+
+* <a class='knowledge-check-link' href='#relations-and-lazy-evaluation'>What are the
+advantages of an Active Record relation over an array?</a>
```

Comment:
While I know that the lesson sort of compares relations to arrays because of the similarity in how they "look" asking for the advantages over an array seems off here since they're really *very* different things.

Additionally, I'm curious why you split this across two lines.